SLA_and_POC.md

```diff
@@ -0,0 +1,11 @@
+# Service Level Agreement (SLA) for KBase
+
+Our normal operating hours are 8am to 7pm Eastern time (7am-6pm Central, 6am-5pm Pacific), Monday through Friday (except holidays).  During these hours, the POC will respond to (not necessarily resolve) production issues within 2 hours.  Outside of these hours, the POC will respond as soon as possible the next business day.  For catastrophic issues that occur outside our normal SLA, the POC will respond as soon as possible.  (Catastrophic issues are those which cause most or all of KBase to be completely unusable with no possible workaround.  Examples include a core service such as workspace or Shock going down.)  Production issues will be routed by the help desk coordinator, internal developers, and the monitoring center to the pager email address.
+
+Issues in non-production environments can be reported to help@kbase.us.  The helpdesk coordinator will route a Jira ticket to the devops component.  These issues will be routed to the most appropriate resource by the POC within one business day.
+
+The POC rotation will be managed in pagerduty. 
+
+The POC will not be assigned a sprint team or other duties during his or her rotation.  The primary duty of the POC will be to respond to reported issues, refer them to other devops team members as needed, and follow up with the reporter to verify a resolution.  Secondary duties may include doing nonintrusive maintenance on KBase systems.
```

Comment:
We probably need some more flexibility here, like on the sprint team but secondary to POC duties.